packages/terra-arrange/tests/jest/Arrange.test.jsx

```diff
@@ -43,16 +43,6 @@ describe('Arrange', () => {
   });
 
   // Prop Tests
-  describe('props validation', () => {
```

Comment:
Removed at @bjankord's instruction, as there exist use-cases in which we shouldn't **force** the user to provide a `fitStart` or `fitEnd`.

cc: @bjankord, @dm123455, @neilpfeiffer, @alex-bezek.